helm/values-uat.yaml

```diff
@@ -178,6 +183,10 @@ microservice-chart:
     OTEL_TRACES_SAMPLER: "always_on"
     ECS_SERVICE_NAME: pagopa-ecommerce-transactions-service
     ECS_SERVICE_ENVIRONMENT: "uat"
+    NPG_URI: "https://localhost/npg"
+    NPG_READ_TIMEOUT: 10000
+    NPG_CONNECTION_TIMEOUT: 10000
+    SPRING_MAX_IN_MEM_SIZE: 16777216
```

Comment:
Why do we need something different from the default value? (262 KB vs 16 MB here)